src/main/java/me/contaria/seedqueue/SeedQueueConfig.java

```diff
@@ -69,6 +69,14 @@ public class SeedQueueConfig implements SpeedrunConfig {
     @Config.Numbers.Whole.Bounds(min = 1, max = 5)
     public int chunkMapScale = 2;
 
+    @Config.Category("chunkmap")
+    public boolean chunkMapShouldFreeze = false;
+
+    @Config.Category("chunkmap")
+    @Config.Numbers.Whole.Bounds(max = Long.MAX_VALUE)
+    @Config.Numbers.TextField
+    public long chunkMapFreezeTime = 160;
+
```

Comment:
I know it's about chunkmaps, but the SeedQueue Chunkmaps category is really only meant for the non-wall chunkmaps. I am considering removing the category completely aswell and just making the option a toggle in debug options only, but thats another discussion.
That being said, the wall category is getting somewhat crowded, maybe it could be added to a misc category (there might be a translation string for that left over from beta).
I'm not sure how much varience there is in chunkmap freeze times people use, but I'd prefer having a slider where -1 is OFF, the upper bound could be cosmetic if people want to set something higher in the config file

Reply:
250 works better for me as opposed to 160 like the original guy had it

idk what category you want it in lol
also there is no leftover translation for any "misc" stuff

Reply:
Maybe just make it a max of 500?
Add it to a new "misc" category with translation "Misc Settings", I'd consider village resetting fairly miscellaneous